exporter/collector/googlemanagedprometheus/naming.go

```diff
@@ -28,6 +28,10 @@ func GetMetricName(baseName string, metric pmetric.Metric) (string, error) {
 	// Second, ad the GMP-specific suffix
 	switch metric.Type() {
 	case pmetric.MetricTypeSum:
+		if !metric.Sum().IsMonotonic() {
+			// Non-monotonic sums are converted to GCM gauges
+			return compliantName + "/gauge", nil
```

Comment:
Sorry one more question, why doesn't this case use `getUnknownMetricSuffix`?

Reply:
Prometheus unknown is always an OTel gauge.  When we "double-write" this gauge, we copy the gauge into a monotonic counter as well, hence the handling above.  There isn't a way to get a non-monotonic sum from an unknown metric point.